docs/shell-integration.md

```diff
@@ -128,6 +128,21 @@ function prompt {
 }
 ```
 
+### OSC 7 on Windows with powershell (with starship)
+
+When you're using [Starship](https://starship.rs/), the prompt is already modify, in this case can add only the necessary output to the already modified prompt, with a function like this in [powershell profile](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_profiles?view=powershell-7.1):
```

Comment:
```suggestion
When using [Starship](https://starship.rs/), since it has taken control of the prompt, hooking in to set
OSC 7 can be achieved this way instead:
```